package.json

```diff
@@ -26,28 +25,16 @@
         "node": ">=10.0.0"
     },
     "homepage": "https://github.com/trailheadapps/lwc-recipes-oss",
-    "husky": {
-        "hooks": {
-            "pre-commit": "lint-staged"
-        }
-    },
     "keywords": [
         "lwc"
     ],
     "license": "CC0-1.0",
-    "lint-staged": {
-        "**/*.{css,html,js,json,yaml,yml,md}": [
-            "prettier --write"
-        ],
-        "**/modules/**/*.js": [
-            "eslint"
-        ]
-    },
     "repository": "trailheadapps/lwc-recipes-oss",
     "scripts": {
         "build": "lwc-services build --mode production",
         "build:development": "lwc-services build",
         "lint": "eslint ./src/**/*.js",
+        "prepare": "husky install",
```

Comment:
Is there a specific reason why you have chosen `prepare`, and not `postinstall`? Because this will also run when I for example publish a package, and there it doesn't make sense.

It would also be good to have a common way to run things across our sample apps, as there are now already [two versions](https://github.com/trailheadapps/easy-spaces-lwc/pull/233/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R18).  @pozil

Reply:
That was proposed by @adityanaag3 but I agree, let's align scripts. I'll make the change.